components/brave_rewards/resources/ui/components/checkout/dialogFrame/assets/dialog_bg.svg

```diff
@@ -0,0 +1,47 @@
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Please use same format of licensing that we have in other files, with `/*`

Reply:
Since this is SVG we'll need `<!--` for comments. Do we want a C-style comment inside of an XML comment?

Reply:
oh totally missed that is svg